### PR TITLE
Fixed logic to open an external url

### PR DESCRIPTION
**Pull request by Phmank**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,11 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/) and this project adheres to [Semantic Versioning](http://semver.org/).
 
+# [1.3.0] - 2020-02-26
```
Comment (Carsten89) [inline CHANGELOG.md]:
```suggestion
# [1.3.1] - 2020-02-26
```
because only bugfixes and no new features

Comment (Carsten89) [inline CHANGELOG.md]:
```suggestion
# [1.2.1] - 2020-02-26
```
because only bugfixes and no new features

**Review by Carsten89 — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (Phmank) [reply CHANGELOG.md]: done (1.2.1)

Comment (Phmank) [reply CHANGELOG.md]: done

**Pull request merged**